### PR TITLE
Widget | Render 404 if story is not found in Storyblok

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Render 404 instead of failing build

- Update how to generate widget landing page paths

## Justify why they are needed

- The build should not fail if the page is not found

- Fix issue where we tried to render "widget/widget" slugs

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3725** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3725?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3725?utm_source=stack-comment).

File: apps/store/src/pages/widget/[[...slug]].tsx [hunk 1]
```diff
@@ -88,31 +90,26 @@ export const getStaticPaths: GetStaticPaths = async (context) => {
...
+
+      return {
+        params: { slug: item.slugParts.slice(1) },
         locale: item.locale,
-      })),
-    fallback: 'blocking',
-  }
-}
+      }
+    })
```
Comment (gustaveen) [inline apps/store/src/pages/widget/[[...slug]].tsx:105]: Could we maybe add a comment/example why we do this? :)

**Review by gustaveen — APPROVED** [review]
Looks good as far as I can see! But I let the build be the final reviewer :D

**Comment by robinandeer** [comment]
### Merge activity

* **Dec 12, 12:23 PM**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3725) rebased this pull request as part of a merge.

**Review by guilhermespopolin — APPROVED** [review]
Noice 🚀

**Pull request merged**